### ISSUE TITLE
People Categories

### ISSUE THREAD

**Issue by walshbr** [scholarslab/scholarslab.org#40]
From @jeremyboggs 
yup, you lose history
we need to figure out a couple of things in the new system
1) What is the person *right now*
2) What has this person been in the past
and manage/display that
like, now, when i remove student-assistant from Christian, it means forever
no difference between current and past
but that’s a known timeframe—an academic year.
probably need like a start date and an end date for each role
basically, a mini CV for SLab-affiliated people.

**Comment by jeremyboggs** [comment]
Updated to remove FUTURE label, since it feels like something we need to address _now_.

**Comment by amandavisconti** [comment]
options for keeping track of current/past staff roles:
1. “current-praxis” and “past-praxis”
2. just have “praxis fellow” and use “status: current” to tell if they’re current or not?
3. or do we want years (e.g. “praxis2018”)

**Comment by amandavisconti** [comment]
Chunking:
staff vs. student, then current vs. former

Description:
Role, Dates
Role2, Dates
Role3, Dates

**Comment by amandavisconti** [comment]
Got the logic working correctly on /people.html! Just need to go through /_people individual files and update them, and also check the logic on the pages listed under "Check" above.

Should share branch with Jeremy and Katherine before PR merge in case it breaks something I'm missing.

**Comment by amandavisconti** [comment]
YAML field work done, /people renders correctly, just need to get individual bio pages rendering correctly for the four person types (former+current staff+student)

**Comment by amandavisconti** [comment]
This is all in the roles-rottweiler branch, btw.
